examples/example-gateway/clients/contacts/fixture/fixture.go

```diff
@@ -15,7 +15,7 @@ var saveContactsFixtures = &mc.SaveContactsScenarios{
 			},
 		},
 
-		Ret0: &gen.SaveContactsResponse{},
+		Ret1: &gen.SaveContactsResponse{},
```

Comment:
Curious why should the `ret0` argument be changed. We're not adding any new args. So there shouldn't be any changes here.

Reply:
This is a fixture to test that the second parameter Ret1 is the correct response, because Ret0 is ctx after the change, and we can not equate that.